EIPS/eip-6189.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 6189
+title: Alias Contracts
+description: Allows the creation of contracts that forward calls to other contracts
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/eip-6190-functional-selfdestruct/12232
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-12-20
+requires: 2929, 6188
+---
+
+## Abstract
+
+This EIP allows contracts to be turned into "alias contracts" using a magic nonce. Alias contracts automatically forward calls to other contracts.
+
+## Motivation
+
+This EIP is not terribly useful on its own, as it adds additional computation and gas costs without any useful side effects. However, it can be used by other EIPs.
```

Comment:
This motivation seems copied from EIP-6188, and probably needs to be updated?

Reply:
Nope. Again, this EIP doesn't do anything useful on its own, but is a useful building block for *other* EIPs. I guess I could point out EIP-6190.